semconv.go

```diff
@@ -1,12 +1,21 @@
 package main
 
 const (
+	// suite keys
 	FailedTestsCount  = "tests.failed"
 	ErrorTestsCount   = "tests.error"
 	PassedTestsCount  = "tests.passed"
 	SkippedTestsCount = "tests.skipped"
 	TestsDuration     = "tests.duration"
-	TestsName         = "tests.name"
-	TestsPackage      = "tests.package"
+	TestsSystemErr    = "tests.systemerr"
```

Comment:
We may need to contribute to OpenTelemtry's `semconv`, as it's very limited from test-results perspective.

@cyrille-leclerc thoughts on that?